handlers/delete_lease.go

```diff
@@ -25,6 +28,21 @@ func DeleteLease(services k8s.ServiceGetterUpdater, k8sServiceName string, names
 			return
 		}
 
+		req := new(api.DeleteLeaseReq)
+		if err := json.NewDecoder(r.Body).Decode(req); err != nil {
+			htp.Error(w, http.StatusBadRequest, "error decoding JSON (%s)", err)
+			return
+		}
+		authToken := os.Getenv("AUTH_TOKEN")
+		if authToken == "" {
+			htp.Error(w, http.StatusInternalServerError, "Server was not started with an auth token.")
+			return
+		}
```

Comment:
same here regarding putting an `AuthToken` field into the config struct & passing it in